WordPressKit/ReaderPostServiceRemote.h

```diff
@@ -62,6 +62,45 @@
                success:(void (^)(RemoteReaderPost *post))success
                failure:(void (^)(NSError *error))failure;
 
+/**
+Fetches the subscription status of the specified post for the current user.
+
+@param postID The ID of the post.
+@param siteID The ID of the site.
+@param success block called on a successful fetch.
+@param failure block called if there is any error. `error` can be any underlying network error.
+*/
+- (void)fetchSubscriptionStatusForPost:(NSUInteger)postID
```

Comment:
We've been trying to write Swift when it comes to new code. Of course, that's a decision that always depends on multiple factors but I think for these new endpoints it makes sense. Wdyt?

Reply:
Fixed! 6e4ec66